www/main.get.php

```diff
@@ -269,21 +269,19 @@
 if (isset($url) && $url) {
     foreach ($inputArguments as $argumentName => $argumentFlag) {
         switch ($argumentName) {
-        case 'limit':
-            if (!is_null($inputGet[$argumentName])) {
-                $centreon->historyLimit[$url] = $inputGet[$argumentName];
-            } elseif (!is_null($inputPost[$argumentName])) {
-                $centreon->historyLimit[$url] = $inputPost[$argumentName];
-            } else {
-                $centreon->historyLimit[$url] = 30;
-            }
-            break;
-
-        default:
-            continue;
-            break;
+            case 'limit':
+                if (!is_null($inputGet[$argumentName])) {
+                    $centreon->historyLimit[$url] = $inputGet[$argumentName];
+                } elseif (!is_null($inputPost[$argumentName])) {
+                    $centreon->historyLimit[$url] = $inputPost[$argumentName];
+                } else {
+                    $centreon->historyLimit[$url] = 30;
+                }
+                break;
+            default:
```

Comment:
remove the default case.
As described in php doc, continue will do exactly the same thing that break in a switch condition ;) https://www.php.net/manual/fr/control-structures.continue.php